portal-frontend/src/app/features/public/search/public-search.component.ts

```diff
@@ -53,6 +60,7 @@ export class PublicSearchComponent implements OnInit, OnDestroy {
 
   notifications: NotificationSearchResultDto[] = [];
   notificationTotal = 0;
+  outcome = '';
```

Comment:
What is this used for? I see it being set here as empty but I don't see it used in the HTML or passed to another component.

Reply:
we use it in mapSearchResults at     this.outcome = searchResult.outcome;

Reply:
Okay, I'm a bit confused on how the main search response has an outcome. I think this is a mistake to add it to SearchResponseDto, as a search result with 10 applications cannot have a single outcome. The outcome should only be present on each result object such as ApplicationSearchResultDto and NoticeOfIntentSearchResultDto.